src/components/API-calls.js

```diff
@@ -75,3 +75,14 @@ export const uploadComment = (review_id, comment) => {
       console.log(err);
     });
 };
+
+export const deleteComment = (comment_id) => {
+  return api
+    .delete(`/api/comments/${comment_id}`)
+    .then((data) => {
+      console.log(data);
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
And this, if it's not doing anything. 
if you're going to have the catch, i'd use it for some error state/component